src/ripple/app/consensus/RCLConsensus.cpp

```diff
@@ -889,12 +903,32 @@ RCLConsensus::Adaptor::onModeChange(ConsensusMode before, ConsensusMode after)
     mode_ = after;
 }
 
+bool
+RCLConsensus::Adaptor::retryAccept(
+    Ledger_t const& newLedger,
+    std::optional<std::chrono::time_point<std::chrono::steady_clock>>& start)
+    const
+{
+    static bool const standalone = ledgerMaster_.standalone();
+    auto const& validLedger = ledgerMaster_.getValidatedLedger();
+
+    return (app_.getOPs().isFull() && !standalone &&
+            (validLedger && (newLedger.id() != validLedger->info().hash) &&
+             (newLedger.seq() >= validLedger->info().seq))) &&
+        (!start ||
+         std::chrono::duration_cast<std::chrono::seconds>(
+             std::chrono::steady_clock::now() - *start)
+                 .count() < 5);
```

Comment:
Avoid escaping the chrono type system.  Safer and simpler:
```c++
        (!start || std::chrono::steady_clock::now() - *start < std::chrono::seconds{5});
```

Reply:
fixed